lib/mix/tasks/phoenix.gen.channel.ex

```diff
@@ -52,6 +52,11 @@ defmodule Mix.Tasks.Phoenix.Gen.Channel do
     unless length(args) == 2 do
       raise_with_help
     end
+
+    args
```

Comment:
Also, the generate module is almost always namespaced and the namespace is not considered here.